data/helpers/postDb.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
       return Promise.all(promises).then(function(results) {
         let [posts, tags] = results;
         let post = posts[0];
-        post.tags = tags.map(t => t.tag);
+        //post.tags = tags.map(t => t.tag);
```

Comment:
I like that you're messing w/ the helpers even though we haven't been instructed on it yet.